plugin/evm/header/extra.go

```diff
@@ -8,10 +8,15 @@ import (
 	"fmt"
 
 	"github.com/ava-labs/coreth/core/types"
-	"github.com/ava-labs/coreth/params"
 	"github.com/ava-labs/coreth/params/extras"
 )
 
+const (
+	// Note: MaximumExtraDataSize has been reduced to 32 in Geth, but is kept the same in Coreth for
+	// backwards compatibility.
+	MaximumExtraDataSize uint64 = 64 // Maximum size extra data may be after Genesis.
```

Comment:
Does this actually make sense to be defined in the `header` package? `header.MaximumExtraDataSize` seems to imply that the `len(ExtraData) <= header.MaximumExtraDataSize` - but that is only true for legacy AP0/launch blocks. Should this be defined in the `ap0` package?